controllers/snapshot/snapshot_adapter.go

```diff
@@ -155,6 +155,10 @@ func (a *Adapter) EnsureAllReleasesExist() (results.OperationResult, error) {
 		return results.ContinueProcessing()
 	}
 
+	if !gitops.IsSnapshotCreatedByPushEvent(a.snapshot) {
+		return results.ContinueProcessing()
```

Comment:
tiny nit: maybe also add an `a.logger.Info` call informing that the snapshot won't be released because it was not created for a push event.

Reply:
Added. Thanks!